nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/test/resources/attachment-only.eml

```diff
@@ -0,0 +1,156 @@
+Date: Fri, 29 Jul 2011 09:41:11 +0200 (CET)
+From: Siegfried GOESCHL <siegfried.goeschl@it20one.at>
+Subject: Kunde 100029   Auftrag   3600
+To: <siegfried.goeschl@it20one.at>
+Reply-To: Siegfried GOESCHL <siegfried.goeschl@it20one.at>
+Message-ID: <ADR36000000000967@it20one.at>
+MIME-Version: 1.0
+Importance: Normal
+X-Priority: 3 (Normal)
+X-Mailer: SAP Web Application Server 7.00
+Content-Type: application/pdf;
+ name="Kunde 100029   Auftrag   3600.pdf"
+Content-Transfer-Encoding: base64
+Content-Description: Kunde 100029   Auftrag   3600

```

Comment:
I would advise just creating your own eml file and submitting that.  It shoudl contain entirely bogus information.  Much simpler than causing license or notice impacts.  In this case we would be pulling from another apache project but it just doesn't seem worth it for a test resource like this so let's just make a new .eml that is fake and nifi only.